fiddler/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Alerting",
+    "Category::Monitoring",
+    "Category::Metrics"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Fiddler",
+    "media": []
+  },
+  "author": {
+    "support_email": "rajeev@fiddler.ai"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "fiddler",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "fiddler.",
+        "check": "",
```

Comment:
Needs a metric to check here. The metric should be one that is always emitted on every check run

Reply:
Done.